release/.buildkite/build_pipeline.py

```diff
@@ -144,6 +144,10 @@ def __init__(self, name: str, retry: int = 0):
         "many_tasks",
         "many_pgs",
         "many_nodes",
+        "many_actors_redis_ha",
```

Comment:
This is dangerous because we might have capacity issue again as all of tests here use many nodes.

What about we create a separate schedule? I can help setting this up

Reply:
So you can add a new mapping here

```
SUITES = {
    "core-nightly": CORE_NIGHTLY_TESTS,
    "serve-nightly": SERVE_NIGHTLY_TESTS,
    "core-daily": CORE_DAILY_TESTS,
    "core-scalability": CORE_SCALABILITY_TESTS_DAILY,
    "nightly": {
        **NIGHTLY_TESTS,
        **USER_TESTS
    },
    "weekly": WEEKLY_TESTS,
}
```
like `core-ha`

and then I can setup the cron job for you after it is merged

Reply:
sure